custom/covid/management/commands/run_all_management_command.py

```diff
@@ -32,16 +32,17 @@ def handle(self, csv_file, **options):
             for row in reader:
                 domains.add(row['domain'])
                 locations = []
-                if row['location1'] != '':
-                    locations.append(row['location1'])
-                if row['location2'] != '':
-                    locations.append(row['location2'])
+                if row['non_traveler_active_location_id'] != '':
+                    locations.append(row['non_traveler_active_location_id'])
+                if row['traveler_active_location_id'] != '':
+                    locations.append(row['traveler_active_location_id'])
                 location_ids[row['domain']] = locations
 
         jobs = []
         pool = Pool(20)
         for domain in domains:
-            jobs.append(pool.spawn(run_command, 'update_case_index_relationship', domain, 'contact'))
+            jobs.append(pool.spawn(run_command, 'update_case_index_relationship', domain, 'contact',
+                                   location=location_ids[domain][1]))
```

Comment:
I'm a little confused by which location this will be. The `!= ''` checks above make me think that neither location is required, and `[domain][1]` could be either the travelers id (if both location ids are provided) or blank (if only one location id was provided). Is that accurate?

Reply:
Nevermind - you updated this in a later commit, I just hadn't seen it yet and couldn't find this comment to delete it.